Makefile

```diff
@@ -38,6 +38,7 @@ docs: submods
 	hugo serve
 
 docs-ci: init submods
+	hugo -s site -b / --gc
```

Comment:
We're setting the baseurl to `/` here so when we do the deploy previews, we dont need to worry about the trailing `/vhs` bit that we need for github pages